dss/error.py

```diff
@@ -81,7 +117,16 @@ def wrapper(*args, **kwargs):
             code = "read_only"
             title = "The DSS is currently read-only"
             stacktrace = ""
-            headers = {'Retry-After': 600}
+            headers = {'Retry-After': '600'}
+
+        # These errors may be returned by the chalice app before reaching here.
+        if status in (requests.codes.server_error,         # 500 status code
```

Comment:
nit. might be useful to have a method that returns this object, used twice so far....